packages/e2e-tests/src/specs/editor/media/insert3PMedia.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { createNewStory } from '@web-stories-wp/e2e-test-utils';
+
+describe('Inserting 3P Media', () => {
+  it('should dismiss message', async () => {
+    await createNewStory();
+
+    const crossOriginIsolated = await page.evaluate(
+      () => window.crossOriginIsolated
+    );
+    expect(crossOriginIsolated).toBeTrue();
```

Comment:
IMO this better fits into `packages/e2e-tests/src/specs/editor/editor.js` in a new separate test.

It is not related to 3P media in any way

Reply:
Moved in https://github.com/google/web-stories-wp/pull/7272/commits/76449cb10808932bf1bd72969c5e5b7d9d3f7800